docs/api/cloudrootfs.md

```diff
@@ -13,44 +13,28 @@ cloud rootfs will package all the dependencies refers to the kubernetes cluster
 │   ├── kubectl
 │   ├── kubelet
 │   ├── nerdctl
+│   ├── kubelet-pre-start.sh
+│   ├── helm
 │   └── seautil
-├── cri
-│   ├── containerd
-│   ├── containerd-shim
-│   ├── containerd-shim-runc-v2
-│   ├── ctr
-│   ├── docker
-│   ├── dockerd
-│   ├── docker-init
-│   ├── docker-proxy
-│   ├── rootlesskit
-│   ├── rootlesskit-docker-proxy
-│   ├── runc
-│   └── vpnkit
-├── etc
+├── etc # configs will be put into all nodes' sealer-rootfs
```

Comment:
What does sealer-rootfs?  Could you help to define that? Since there are quite a lot of rootfs in sealer's concept. @VinceCui